### PR TITLE
Add support for build and push docker image

### PR DESCRIPTION
**Pull request by eddumelendez**

### CODE REVIEW

File: spring-cloud-schema-registry-server/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM java:8-alpine
+
+ARG JAR_FILE
+
+ADD target/${JAR_FILE} spring-cloud-skipper-server.jar
```
Comment (tzolov) [inline spring-cloud-schema-registry-server/Dockerfile]: I guess you've copied the Dockerfile from the skipper project but forgot to change the jar names. 
Replace the `spring-cloud-skipper-server.jar` by `spring-cloud-schema-registry.jar` instead.

File: spring-cloud-schema-registry-server/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+ENTRYPOINT exec java -Djava.security.egd=file:/dev/./urandom -jar /spring-cloud-skipper-server.jar
```
Comment (tzolov) [inline spring-cloud-schema-registry-server/Dockerfile]: Again replace the `spring-cloud-skipper-server.jar` by `spring-cloud-schema-registry.jar` instead.

File: spring-cloud-schema-registry-server/pom.xml [hunk 3]
```diff
@@ -43,6 +47,18 @@
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>com.spotify</groupId>
+                <artifactId>dockerfile-maven-plugin</artifactId>
+                <version>1.4.13</version>
```
Comment (tzolov) [inline spring-cloud-schema-registry-server/pom.xml]: Replace the hardcoded version with a property (e.g. `<version>${dockerfile-maven-plugin.version}</version>`) and define the `dockerfile-maven-plugin.version` property in the [parent's POM property section](https://github.com/spring-cloud/spring-cloud-schema-registry/blob/master/pom.xml#L24-L30)

**Comment by eddumelendez** [comment]
thanks @tzolov ! I just pushed all changes needed

**Comment by tzolov** [comment]
Thanks @eddumelendez!
PR is rebased and merged at 351f9cbc00515416ed9b9d8927649e8b08bc8d6c

**Pull request closed without merging**